local-cli/link/windows/patches/makeProjectPatch.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function makeProjectPatch(projectInsert, projectName = '') {
+  return {
+    pattern: '<ProjectReference Include="..\\..\\node_modules\\react-native-windows\\ReactWindows\\ReactNative\\ReactNative.csproj">',
```

Comment:
>ProjectReference [](start = 15, length = 16)

I don't think this works.  This is typically followed by:
```
      <Project>{c7673ad5-e3aa-468c-a5fd-fa38154e205c}</Project>
      <Name>ReactNative</Name>
    </ProjectReference>
```
And the full project reference should come after the close </ProjectReference>